challenge/src/components/Pages/Contact.jsx

```diff
@@ -10,44 +11,47 @@ const Contact = (props) => {
         marginLeft: "5%"
     }
     return (
-        <div>
+        <>
+          <div>
             <div>
-            <h1 style={{ fontSize: 100, color: "#F09F70" }}>
-              <center>Contact Page</center>
-            </h1>
-            <div style={{ backgroundColor: "whitesmoke" }}>
-              <br />
-            </div>
-            <div
-              style={{
-                backgroundColor: "#546882",
-                display: "flex",
-                height: "100%",
-                alignItems: "center",
-                justifyContent: "center",
-                flexDirection: "row"
-              }}
-            >
-              <RandomImage
-                page={page * 4}
-                style={pictureStyling}
-              />
-              <RandomImage
-                page={page * 8}
-                style={pictureStyling}
-              />
+              <h1 style={{ fontSize: 100, color: "#F09F70" }}>
+                <center>Contact Page</center>
+              </h1>
+              <div style={{ backgroundColor: "whitesmoke" }}>
+                <br />
+              </div>
```

Comment:
please avoid adding formatting changes or reindenting code to PRs that also include other stuff. it makes it hard to tell what things actually change during code review. this should be done in separate PRs

Reply:
Gotcha. Indented everything in a level because I wrapped everything with a `<>`.